src/main/java/javax/validation/Configuration.java

```diff
@@ -234,6 +249,20 @@
 	ParameterNameProvider getDefaultParameterNameProvider();
 
 	/**
+	 * Returns an implementation of the {@link ClockProvider}
+	 * interface following the default {@code ClockProvider}
+	 * defined in the specification:
+	 *
+	 * XXX BVAL-496 paste definition of the specification
```

Comment:
About TODO vs XXX, the idea is that XXX is something that really needs to be fixed (in my mind, it needs to be fixed before the release and nothing should be released with an XXX in it). A TODO might have a longer lifecycle and is less imperative. They are both supported by Eclipse.